app.rb

```diff
@@ -0,0 +1,33 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each_with_index { |book, n| print "(#{n}) Title: \"#{book.title}\", Author: \"#{book.author}\"\n" }
+  end
+
+  def list_people
+    # people = [*@teachers, *@students]
+    @people.each_with_index do |person, i|
+      puts "(#{i}) #{person.is_a?(Student) ? '[Student]' : '[Teacher]'} Name: \"#{person.name}\", Age: \"#{person.age}"
```

Comment:
- [ ] **Optional:** Consider making public the id of a person as this is used to show a rental and is currently inaccessible.

Reply:
Great job and brilliant idea
thanks we will definitely work on that now .